sound/soc/intel/boards/skl_hda_dsp_generic.c

```diff
@@ -206,7 +228,12 @@ static int skl_hda_audio_probe(struct platform_device *pdev)
 		hda_soc_card.components = hda_soc_components;
 	}
 
-	return devm_snd_soc_register_card(&pdev->dev, &hda_soc_card);
+	ret = devm_snd_soc_register_card(&pdev->dev, &hda_soc_card);
+
```

Comment:
there's usually no empty line in sequences like
```
ret = fn();
if (ret < 0)
```

Reply:
OK, will change it.